modular_zubbers/code/modules/mob/living/carbon/examine.dm

```diff
@@ -0,0 +1,6 @@
+/atom/proc/examine_title_worn(mob/user)
+	var/regular_examine = src.examine_title(user)
+	if(HAS_TRAIT_FROM(src, TRAIT_WAS_RENAMED, "Loadout")) // Uses /datum/element/examined_when_worn
```

Comment:
It would be better if /datum/element/examined_when_worn gives the trait that examine_title_worn checks if it can examine, so the element can be nicely used on it's own instead of needing external implementation

Reply:
Done